cmd/internal/apidump/apidump.go

```diff
@@ -192,40 +190,20 @@ var Cmd = &cobra.Command{
 }
 
 func init() {
-	Cmd.Flags().Var(
```

Comment:
Removed `--out`, `--service` and `--cluster` commands.

Can't completely remove the `outFlag` var since it might need some major refactoring. outFlag will be `nil` now.